node/heartbeat/export_test.go

```diff
@@ -70,6 +70,18 @@ func (hbmi *heartbeatMessageInfo) GetTotalDownTime() time.Duration {
 	return hbmi.totalDownTime
 }
 
+func (hbmi *heartbeatMessageInfo) GetComputedShardId() uint32 {
```

Comment:
Missing comments

Reply:
it,s in export_test. we didn't comment these methods until now. but fixed

Reply:
yeah, saw that now. Will try not ask this in the future :D